### DISCUSSION TITLE
Add date type

### DISCUSSION THREAD

**Discussion by fkrauthan** [keystonejs/keystone#7473] [Feature requests]
It would be great to have a Date only type (instead of timestamp). If we need to stay compatible with GraphQL we could accept a full timestamp and always throw away the time part and when sending data always set the Time portion as `00:00:00`. But since some databases support a Date only field and there are use-cases where a timestamp is not what you want having that capability would be great.

**Comment by dcousens** [comment]
Hi @fkrauthan,
A new field for  `calendarDay` is being added in https://github.com/keystonejs/keystone/pull/7658 :rocket::date:
